gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/model/Subscription.java

```diff
@@ -105,6 +115,8 @@ public enum AuditEvent implements Audit.ApiAuditEvent {
 
     private Date pausedAt;
 
+    private Date consumerPausedAt;
```

Comment:
it's in relation with the previous message, but the idea behind that is to provide information about when the subscription was paused and by who.
It could be useful in the UI to know that, even with consumer resume, subscription is STILL paused because of the publisher, and reverse is also true.